content/02-types.md

```diff
@@ -308,6 +308,123 @@ Like classes, interfaces can be marked with the `final` keyword, preventing them
 
 
 
+<!--label:types-abstract-class-->
+#### Abstract Class
+
+##### since Haxe 4.2.0
+
+Abstract classes (not to be confused with [Abstract](type-system-type-inference)) are classes with partial implementation. As such, an abstract class cannot be instantiated but must first be extended, with the child class providing implementation of any abstract methods. Abstract classes support all language features that concrete classes support, thus any class can be declared abstract.
```

Comment:
We should say "with the child class providing implementation of any abstract methods or being declared abstract itself".